.gitignore

```diff
@@ -158,3 +158,8 @@ test/input/*
 !test/input/.keep
 test/output/*
 !test/output/.keep
+
+# input
+input/*
+!input/owl-files/
```

Comment:
## Where to put ChEBI inputs
I have the goal of moving all of the sources somewhere in `input/`. I've decided that'll probably be `input/sources/`, or something to denote all of the inputs that actually go in to generating our outputs.

As I'm not sure yet if ChEBI will be such an input, I'm putting it in `input/analysis/`.

I also plan to eventually move `owl-files/` and `data/` into `input/`, but won't do that in this PR.